documentation/questionnaire.txt

```diff
@@ -0,0 +1,2 @@
+This text contains link for the questionnaire made on Google Forms, whith intention of develop user profile and personas.
```

Comment:
I think it would be better if we addressed it this way:

[This is the link](https://goo.gl/forms/Wkl1psJQlDE21W4N2) for the questionnaire made with Google Forms, with the intention of assisting us in the development of our user profile and persona

This is using Markdown syntax. I think it would be better if we make all simple documents on the project using `.md` files, since it would be easier for us to format and read them on GitHub.